tests/components/switchbot_cloud/test_lock.py

```diff
@@ -33,7 +38,7 @@ async def test_lock(hass: HomeAssistant, mock_list_devices, mock_get_status) ->
     assert entry.state is ConfigEntryState.LOADED
 
     lock_id = "lock.lock_1"
-    assert hass.states.get(lock_id).state == LockState.LOCKED
+    assert hass.states.get(lock_id).state == STATE_UNKNOWN
```

Comment:
I was expecting to see a change the other way around. Why is it now `unknown` instead of using the value provided in `mock_get_status`?

Reply:
You're right, it doesn't make sense. Due to the way it was working previously, the initial coordinator refresh wasn't being processed. It should now be fixed